conf.py

```diff
@@ -65,7 +65,7 @@
         "resource for web development are the MDN web docs: http://developer.mozilla.org/en-US/docs/Learn"
     ),
     "howto-ask": (
-        "Asking questions well increases your chance of getting help. "
+        "Asking questions will increase your chance of getting help. "
```

Comment:
"well" as in, "good".

"will" doesn't make a ton of sense in this context. Asking vs not asking will increase your chance?

The intent here is that "Asking questions which are formulated _well_" vs "Asking poorly formulated questions".

Reply:
Oh, I completely misunderstood that message. I failed to parse it as "asking questions in a good way".
Sorry.

Reply:
No worries! I'm open to a PR that makes the language more clear.